cockatrice/src/tab_game.cpp

```diff
@@ -434,12 +434,14 @@ void TabGame::emitUserEvent() {
 
 TabGame::~TabGame()
 {
-    if(replay)
-        delete replay;
+    delete replay;
```

Comment:
Why has this check been removed?

Reply:
Clion informed me that "delete Object" and "delete nullptr" are both fine, so the check is superfluous

Reply:
Yep, c++11, you are right.